rosbag2_transport/src/rosbag2_transport/player.cpp

```diff
@@ -83,19 +83,39 @@ rclcpp::QoS publisher_qos_for_topic(
   const auto offered_qos_profiles = profiles_yaml.as<std::vector<Rosbag2QoS>>();
   return Rosbag2QoS::adapt_offer_to_recorded_offers(topic.name, offered_qos_profiles);
 }
+
 }  // namespace
 
 namespace rosbag2_transport
 {
 constexpr Player::callback_handle_t Player::invalid_callback_handle;
 
+Player::Player(const rclcpp::NodeOptions & node_options)
+: Player("rosbag2_player", node_options) {}
```

Comment:
Just want to point at  https://github.com/ros2/rosbag2/pull/1432. Users creating these nodes compositionally should be able to name the node explicitly, does this  cause any issues for that? I'm not sure if so or not